ui/app/components/line-chart.js

```diff
@@ -1,23 +1,16 @@
-/* eslint-disable ember/no-observers */
```

Comment:
Nice to be free of an observer!

Reply:
What was especially nice is I didn't have to replace it with anything. As a product of refactoring everything and auto-tracking it Just Worked™.